internal/handlers/common/filter.go

```diff
@@ -468,6 +487,13 @@ func filterFieldExpr(doc *types.Document, filterKey string, expr *types.Document
 				return false, err
 			}
 
+		case "$all":
+			// {field: {$all: [value, another_value, ...]}}
+			res, err := filterFieldExprAll(fieldValue, exprValue)
+			if !res || err != nil {
```

Comment:
can it be replaced with?
```go
return filterFieldExprAll(fieldValue, exprValue)
```

Reply:
None of the other operators return it like this.

Reply:
> None of the other operators return it like this.

The question is equal to - if it can be used with other operators? 
If yes, then let's add a test with another operator. 
If no, then it can be returned.

Reply:
That seems to be out of the scope of the current task. If you think this approach needs to be refactored, please create a separate issue.